dbt_meshify/dbt_projects.py

```diff
@@ -432,6 +459,39 @@ def _get_custom_macros(self) -> Set[str]:
                 macros_set.update(self._get_macro_dependencies(macro))
         return macros_set
 
+    @property
+    def referenced_docs(self) -> Set[str]:
+        """Return a list of all docs referenced within this SubProject."""
+
+        if self._referenced_docs:
+            return self._referenced_docs
+
+        docs = set()
+        for unique_id in self.resources:
+            if unique_id.startswith("test."):
+                continue
+
+            node = self.get_manifest_node(unique_id)
+
+            if node is None:
```

Comment:
should this throw an error? i think we throw a similar one if we can't find a node in the manifest elsewhere